src/preset-themes/008.json

```diff
@@ -2,44 +2,44 @@
   "title": "008",
   "colors": {
     "toolbar": {
-      "h": 120,
-      "s": 1,
-      "l": 86,
-      "a": 100
+      "r": 184,
+      "g": 255,
```

Comment:
So this color is way wring...i think something's happening where 1 is being converted to 100% saturation by tinycolor?

Reply:
I guess i commented the wrong line, the old saturation value here is 1 which may be getting converted to 100% somewhere in tinycolor before it gets converted to rgb?

Reply:
Yeah, according to http://hslpicker.com/#dbdcdb, this should be `{r: 219, g: 220, b: 219, a: 1}`